logger/logger.go

```diff
@@ -0,0 +1,78 @@
+// Package logger log data to Stderr and file.
+package logger
+
+import (
+	"fmt"
+	"io"
+	"log"
+	"os"
+	"path/filepath"
+	"time"
+)
+
+var loggingFiles []io.WriteCloser
+
+// Logger has information for logging
+type Logger struct {
+	File  io.WriteCloser
+	Debug *log.Logger // Debug is for debug log. You can set log flag. Also you can choose log stderr or not
+	Error *log.Logger // Error is always debug file and stderr with LstdFlags flag.
+}
+
+// New returns logger object
+func New(dirPath, filename string, flag int, debug bool) (*Logger, error) {
+	lagger := new(Logger)
```

Comment:
typo `lagger`

Reply:
Actually this is a difficult to set name. We import logger thus it is not good use logger for variable name. Also use log is not good because there is standard library log.

Some library(https://github.com/cloudfoundry/lager) use lager for logger, so I imitate the name. 
Even though I missed name `lager` to `lagger`.
I will fix.

Reply:
Or if you have any good idea for variable name please tell me.

Reply:
Oh! How about `lgr`?

Reply:
`lgr` looks better. I choose `lgr`.